cal-itp/benefits
# Feat: create a separate eligibility start page

This closes #332.

## Testing this PR

1. Delete your local database and compiled language files:

  ```bash
  rm django.db
  
  rm benefits/locale/**/*.mo
  ```

2. Reinitialize

  ```bash
  bin/init.sh
  ```

3. Launch the app with `F5`, navigate to the homepage, and select an agency.
4. Manually enter the url `/eligibility/start` and you should see the same page/content as `/eligibility`:

`/eligibility`:

![image](https://user-images.githubusercontent.com/1783439/159526749-9ca19fe1-e511-4219-87a2-48714084e1d2.png)

`/eligibility/start`:

![image](https://user-images.githubusercontent.com/1783439/159526883-47750377-a065-4ae1-a647-aa11d2557fa7.png)
